models/advanced-home-example/IRoom.json

```diff
@@ -32,7 +32,7 @@
     "displayName": "thermostat component",
     "contents": [
       {
-        "@type": ["Property", "Temperature"],
+        "@type": ["Telemetry", "Temperature"],
```

Comment:
@danhellem  Based on your comment that telemetry doesn't make so much sense on IHome, is the same true for IRoom? Should I remove it here too?

Reply:
@baanders well the component is used by the room. So lets keep that a property and only change the sensor to use telemetry